apps/wing/src/commands/compile.ts

```diff
@@ -234,40 +234,41 @@ export async function compile(entrypoint: string, options: CompileOptions): Prom
   try {
     vm.runInContext(artifact, context);
   } catch (e: any) {
-    console.error(e.message);
+    const output = new Array<string>();
 
-    if (process.env.DEBUG) {
-      if ((e as any).stack && (e as any).stack.includes("evalmachine.<anonymous>:")) {
-        console.log();
-        console.log(
-          "  " +
-            chalk.bold.white("note:") +
-            " " +
-            chalk.white(`intermediate javascript code (${artifactPath}):`)
-        );
-        const lineNumber =
-          Number.parseInt((e as any).stack.split("evalmachine.<anonymous>:")[1].split(":")[0]) - 1;
-        const lines = artifact.split("\n");
-        let startLine = Math.max(lineNumber - 2, 0);
-        let finishLine = Math.min(lineNumber + 2, lines.length - 1);
-
-        // print line and its surrounding lines
-        for (let i = startLine; i <= finishLine; i++) {
-          if (i === lineNumber) {
-            console.log(chalk.bold.red(">> ") + chalk.red(lines[i]));
-          } else {
-            console.log("   " + chalk.dim(lines[i]));
-          }
+    output.push(chalk.red(`ERROR: ${e.message}`));
+
+    if ((e as any).stack && (e as any).stack.includes("evalmachine.<anonymous>:")) {
+      const lineNumber =
+        Number.parseInt((e as any).stack.split("evalmachine.<anonymous>:")[1].split(":")[0]) - 1;
+      
+      output.push('');
+      output.push(chalk.underline(chalk.dim(`${artifactPath}:${lineNumber}`)));
```

Comment:
Could this artifactPath instead be made relative to cwd?

Reply:
Done